CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 2.0.5
+
+* Enhancement of the support for reflection on type arguments, covering
+  cases where a type annotation is a parameterized type where all type
+  arguments are resolved statically (e.g., `List<int> f();`). We (still) do
+  not support cases where one or more type arguments are or contain
+  type variables from an enclosing function or class (like the type of
+  the formal parameter to `List.addAll`: `List<E>`), and with such type
+  arguments a dynamic error is raised if an attempt is made to access it.
+  Similarly, we (still) do not support extracting the value of the actual
```

Comment:
does the `(still)` mean that we are going to support it at some point?

Reply:
I wanted to remind the reader that we did not previously support the feature in question, as opposed to 'with this update we stopped supporting that feature'.

Another thing is that I do expect some level of support for extracting the actual type arguments "at" some specific type (that is, I expect that we will get some kind of reified existential open), so in that sense it's not unreasonable to read it as "we hope to get it later".

So, trying to come up with something better, I think I'll just step back and leave it as is. ;)